### PR TITLE
:tada: RMRK 2.0 beta

### PR DESCRIPTION
**Pull request by vikiival**
- :arrow_up: @kodadot1/vuex-options
- :zap: enable RMRK2
- :arrow_up: @kodadot1/vuex-options
- :wrench: rmrk2 config
- :arrow_up: @kodadot1/vuex-options
- :tada: RMRK2

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [ ] Closes #<issue_number>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### CODE REVIEW

File: pages/rmrk2/index.vue [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    return 'full-width-layout'
+  }
+
+  middleware({ store, redirect }) {
+    // If the user is not authenticated
+    const prefix = store.getters.currentUrlPrefix
+    if (prefix !== 'movr') {
+      this.$consola.log('Not Moonriver')
+      store.dispatch('setUrlPrefix', 'movr')
+    }
```
Comment (roiLeo) [inline pages/rmrk2/index.vue:18]:
```suggestion
    if (prefix !== 'rmrk2') {
      this.$consola.log('Not RMRK2')
      store.dispatch('setUrlPrefix', 'rmrk2')
    }
```

File: pages/rmrk2/index.vue [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export default class LandingPage extends Vue {
+  layout() {
+    return 'full-width-layout'
+  }
+
+  middleware({ store, redirect }) {
+    // If the user is not authenticated
+    const prefix = store.getters.currentUrlPrefix
+    if (prefix !== 'movr') {
+      this.$consola.log('Not Moonriver')
```
Comment (Jarsen136) [inline pages/rmrk2/index.vue:16]: 👀 

image

**Review by Jarsen136 — CHANGES_REQUESTED** [review]

**Review by roiLeo — COMMENTED** [review]
doesn't work for me
Screenshot 2023-02-16 at 16-31-26 KodaDot - NFT Market Explorer

Some nftEntity name are `null`
collectionEntities name are `null` too

**Comment by vikiival** [comment]
> 
> Some nftEntity name are `null`
> 
> collectionEntities name are `null` too
> 
> 

Yes, infura had some outage while indexing. 

See kodadot/workers#32

**Comment by yangwao** [comment]
> infura had some outage while indexing.

https://status.infura.io/ seems only for Polygon today?

can we switch to Pinata for indexer as we have there 100k req which should sufficient for indexing I guess?

image

**Comment by yangwao** [comment]
fleek, cf-ipfs or someone from this list? https://ipfs.github.io/public-gateway-checker/

**Comment by yangwao** [comment]
ok, for indexing we might have ipfs gateway, seems it's infura costs are $200 for what we've been paying at pinata $1k, but with r2 even lower, let's see

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
seems it's working... partially? 
- https://deploy-preview-5028--koda-nuxt.netlify.app/rmrk2/gallery/15517161-90c6619c6b94fcfd34-EVRLOOT_RESOURCES-WATER-11?page=1

image

**Comment by yangwao** [comment]
I think we will have issues with numbers, not sure if they are right? 
https://deploy-preview-5028--koda-nuxt.netlify.app/rmrk2/collection/90c6619c6b94fcfd34-EVRLOOT_RESOURCES?page=1

image

**Comment by yangwao** [comment]
Different currencies on cards

image

**Comment by yangwao** [comment]
seems stil syncing
- https://squiki.kodadot.workers.dev/status/rubick

```json
"name": "009-rc0",
"squidName": "rubick",
"status": "SYNCING",
"syncStatus": {
"totalBlocks": 16673334,
"currentBlock": 15517829
```

**Comment by yangwao** [comment]
Ok, my proposal would be to merge it partially it's working and there needs to be done lot of fixes beyond this PR 

- #3725

**Comment by vikiival** [comment]
>  or hit our image-workers? I think it better to hit our image-workers

using minipfs

https://github.com/kodadot/rubick/blob/zlozenec/src/mappings/utils/metadata.ts

**Comment by yangwao** [comment]
Let's push it on beta :)

**Comment by yangwao** [comment]
now?

**Pull request merged**